fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java

```diff
@@ -136,4 +143,63 @@ public List<String> listStorageVolumeNames() throws DdlException {
                     .stream().map(FileStoreInfo::getFsName).collect(Collectors.toList());
         }
     }
+
+    public void createOrUpdateBuiltinStorageVolume() throws DdlException, AnalysisException, AlreadyExistsException {
+        if (Config.cloud_native_storage_type.isEmpty()) {
+            return;
+        }
+
+        List<String> locations = parseLocationsFromConfig();
+        Map<String, String> params = parseParamsFromConfig();
+
+        try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
+            if (exists(BUILTIN_STORAGE_VOLUME)) {
+                updateStorageVolume(BUILTIN_STORAGE_VOLUME, params, Optional.empty(), "");
+            } else {
+                createStorageVolume(BUILTIN_STORAGE_VOLUME,
```

Comment:
do we have parameter validation on starmgr side? e.g create a s3 filestore without region, ak/sk, I am afraid of incomplete configuration provided to filestore while the creation doesn't discover it and still create successful.

Reply:
It is not implemented yet.